src/main/java/core/basesyntax/garage/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.garage;
+
+public class Excavator extends Machine{
+    public void doWork() {
```

Comment:
The method 'doWork' should provide an informative message indicating that it is an 'Excavator' that is working. Consider including the type of machine in the message.